api/v1alpha1/common.go

```diff
@@ -244,6 +244,30 @@ const (
 	DataProcessType OperationType = "DataProcess"
 )
 
+// AffinityPolicy the strategy for the affinity between Data Operation Pods.
+type AffinityPolicy string
+
+const (
+	DefaultAffinityStrategy AffinityPolicy = ""
+	RequireAffinityStrategy AffinityPolicy = "Require"
+	PreferAffinityStrategy  AffinityPolicy = "Prefer"
+)
+
+type AffinityStrategy struct {
+	// Policy one of: "", "Require", "Prefer"
+	// +optional
+	Policy AffinityPolicy `json:"policy,omitempty"`
+
+	Prefer  []Prefer `json:"prefer,omitempty"`
+	Require []string `json:"require,omitempty"`
```

Comment:
How about change the type of `Require` as another type struct for future extensibility?

```
Require []Require

type Require struct {
    name string
}
```

Reply:
fixed.